css/index.css

```diff
@@ -56,6 +61,33 @@ html, body {
     font-family: 'Roboto', sans-serif;
 }
 
+/*======================
+  Nav and Header Styles
+========================*/
```

Comment:
Yesssssss, you love to see it. Comments that describe and are organized with their respective classes down below. Great job, keep that good habit up.